src/constants.js

```diff
@@ -10,7 +10,7 @@ export const DEFAULT_DESCRIPTOR = 'lowlevel.mfcc.mean';
 export const PERFORM_QUERY_AT_MOUNT = false;
 
 // backend and login urls
-const BACKEND_APPLICATION_ROOT = '/fse';  // must match with backend's application root
+const BACKEND_APPLICATION_ROOT = '/fse/backend' // -> must match with backend's application root
```

Comment:
Don't understand why you made this change here, I think we should keep `/fse` or move this to some option that can be configured elsewhere. The deployed version of FSE uses /fse`.

Reply:
I can not remember - had some trouble with saving and loading in the beginning and changed some paths. Just ignore if it works that way.

Reply:
I'd say make sure that you can make it work reverting that change and commit again in this PR. Then we can merge.